src/win/tty.c

```diff
@@ -1616,6 +1630,24 @@ static int uv_tty_write_bufs(uv_tty_t* handle,
     uv_buf_t buf = bufs[i];
     unsigned int j;
 
+  if (uv__vterm_state == UV_SUPPORTED) {
+    utf16_buf_used = MultiByteToWideChar(CP_UTF8, 0, buf.base, buf.len, NULL, 0);
+
+    if (utf16_buf_used == 0) {
+      *error = GetLastError();
+      break;
+    }
+
+    if (!MultiByteToWideChar(CP_UTF8, 0, buf.base, buf.len,
+                             utf16_buf, utf16_buf_used)) {
```

Comment:
Just to clarify, do you mean like this? (for the calls to `MultiByteToWideChar`)

```
MultiByteToWideChar(CP_UTF8,
                    0,
                    buf.base,
                    buf.len,
                    utf16_buf,
                    utf16_buf_used)
```